test/UnbrandedProjects/NoTest-TypeSpec/src/Generated/Configuration.json

```diff
@@ -0,0 +1,14 @@
+{
+  "output-folder": ".",
+  "namespace": "NoTestTypeSpec",
+  "library-name": "NoTestTypeSpec",
+  "shared-source-folders": [
+    "../../../../../artifacts/bin/AutoRest.CSharp/Debug/net6.0/Generator.Shared",
+    "../../../../../artifacts/bin/AutoRest.CSharp/Debug/net6.0/Azure.Core.Shared"
+  ],
+  "use-overloads-between-protocol-and-convenience": true,
```

Comment:
For the unbranded project, I did not see any xml for docs generated, is this expected?

Reply:
Yes we don't want any docs only 1 single smoke test unless you turn that off which is in the NoTest-TypeSpec project